README.md

```diff
@@ -66,6 +66,15 @@ We then construct a main export file that knows how to parse `next.config.js`
 and route to the correct files, which we've statically analyzed and are included
 in the generated `dist/index.js` file.
 
+## Caveats
+
+Vercel's Edge Runtime is supported only when running in a Node.js environment. In other words, a bundled nsm project cannot be run in the Edge Runtime.
```

Comment:
Hmm this isnt a fundamental limitation right? We could use eg an edge fake inside seam connect and run it on edge?

Reply:
I don't think it's a fundamental limitation, but just wanted to limit scope in this initial PR

Reply:
yep i agree